cl/scrapers/tasks.py

```diff
@@ -141,16 +141,17 @@ def extract_from_pdf(path, opinion, do_ocr=False):
     """
     process = make_pdftotext_process(path)
     content, err = process.communicate()
+    content = content.decode()
     if content.strip() == "" and do_ocr:
         success, content = extract_by_ocr(path)
+        content = content.decode()
```

Comment:
@flooie can you explain what's happening with these two decodes? It looks wrong to me, but maybe I'm missing something.